pkgs/development/tools/godot/3/mono/make-deps.nix

```diff
@@ -0,0 +1,39 @@
+{ godot-mono, nuget-to-nix }:
+
+godot-mono.overrideAttrs (self: base: {
+  pname = "godot-mono-make-deps";
+
+  nativeBuildInputs = base.nativeBuildInputs ++ [ nuget-to-nix ];
+
+  shouldConfigureNuget = false;
+
+  outputs = [ "out" ];
+  buildPhase = " ";
+  installPhase = ''echo "No output intended. Run make-deps.sh instead." > $out'';
+
+  makeDeps = ''
+    set -e
+    outdir="$(pwd)"
+    wrkdir="$(mktemp -d)"
+    pushd "$wrkdir" > /dev/null
+      unpackPhase
+      cd source
+      patchPhase
+      configurePhase
+
+      # Without RestorePackagesPath set, it restores packages to a temp directory. Specifying
+      # a path ensures we have a place to run nuget-to-nix.
+      nugetRestore() { dotnet msbuild -t:Restore -p:RestorePackagesPath=nugetPackages $1; }
+
+      nugetRestore modules/mono/glue/GodotSharp/GodotSharp.sln
+      nugetRestore modules/mono/editor/GodotTools/GodotTools.sln
+
+      nuget-to-nix nugetPackages > "$outdir"/deps.nix
+    popd > /dev/null
+    rm -rf "$wrkdir"
+  '';
+
+  meta = base.meta // {
+    description = "A derivation with no output that exists to provide an environment for make-deps.sh";
```

Comment:
```suggestion
    description = "Derivation with no output that exists to provide an environment for make-deps.sh";
```

Reply:
I can't respond directly to your "Does mono not have an inline updater which just needs to be added to passthru?" question for some reason, so I'll respond here:  I'm not sure.  I tinkered with this for a while trying to get something that worked and this is what I landed on.  I was aware of existing nuget-depedency updaters, but for some reason I concluded they wouldn't work in this case.  It's been a while though, so I don't remember the details.  And maybe I missed something.  I'd rather not redo this work unless the solution is straightforward.